.github/workflows/wheel_macOS.yml

```diff
@@ -0,0 +1,57 @@
+name: wheel - macOS
+
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  wheel_macos:
+    name: macOS wheel
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest]
+        python-version: [3.7, 3.8]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: check C++ compiler and Python
+        run: |
+          clang --version
+          clang++ --version
+          which clang
+          python --version
+          which python
+      - name: Install dependencies
+        run: |
+          python -m pip install --no-cache-dir --upgrade pip
+          python -m pip install --no-cache-dir "poetry<2,>=1.0"
+          python -m pip install --no-cache-dir "mypy-protobuf~=1.0"
+      - name: Build libraries
+        run: |
+          make
+      - name: Build wheel
+        run: |
+          python -m poetry build -v
+          python -m pip install --no-cache-dir dist/*.whl
+      - name: Tests
+        env:
+          PYTEST_ARGS: "--forked"
+        if: success()
+        run: |
+          # export test dependencies from pyproject.toml, install them
+          python -m poetry export -f requirements.txt --dev -o requirements.txt && pip install -r requirements.txt
+          mkdir testdir && cd testdir
+          poetry run pytest -s -v --cov=httpstan --cov-fail-under=93 $PYTEST_ARGS ../tests
+          cd ..
+      - name: Upload to pypi
+        env:
+          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
```

Comment:
No need for PyPI username, we can use project-specific, scoped API tokens, https://pypi.org/help/#apitoken

Reply:
I'm doing it, it is just under the secrects.PYPI_USERNAME

Reply:
Twine username can be a string `__token__`, right? That means we don't need a variable. Or have I misread the instructions?

Reply:
Yes, I just put it under secrets, we can pull it out and use bare `__token__`

Reply:
It's not conventional to make a variable for a constant string value, is it? Or am I missing something?

Reply:
maybe not

Reply:
changed to `__token__`